src/armature/CostFunction.ts

```diff
@@ -0,0 +1,182 @@
+import { coord } from '../calder';
+import { AABB } from '../geometry/BakedGeometry';
```

Comment:
I'm not the biggest fan of the `AABB` name. Is there something more descriptive we could use?

Reply:
Nevermind, i'm wilin out